tests/disas/winch/aarch64/call/simple.wat

```diff
@@ -0,0 +1,40 @@
+;;! target = "aarch64"
+;;! test = "compile"
```

Comment:
I'm not entirely sure if this is the source of the failure, but here you need to specify 

```suggestion
;;! test = "winch"
```

Else this will use Cranelift

Reply:
Some submodule update sneaked into one of my commits. That's what's messing with the CI, I'll fix that tommorow